akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -95,7 +95,11 @@ akka.cluster.sharding {
   least-shard-allocation-strategy {
     # Threshold of how large the difference between most and least number of
     # allocated shards must be to begin the rebalancing.
-    rebalance-threshold = 10
+    # 1 gives the best distribution and therefore typically the best choice.
+    # Increasing the threshold can result in quicker rebalance but has the
+    # drawback of increased difference between number of shards (and therefore load)
+    # on different nodes before rebalance will occur.
+    rebalance-threshold = 1
```

Comment:
I agree with the nervousness about the drastic change.

Is there any risk of continual rebalance with the threshold set to 1?

I.E. in an uneven sharding distribution, is there any risk that a shard will be rebalanced to another node, only to be immediately rebalanced back again (because it's impossible to remove the uneven distribution)? Or even if we don't rebalance back is there any risk of wasting CPU or network resources trying to figure out if we should rebalance because we will always be meeting the minimum threshold?

Reply:
Disregard my previous comment. I see below that it has to be greater than the threshold in order to trigger a rebalance.

Should the comment above `rebalance-threshold` indicate that it must be greater than?

Reply:
added